cosmos/scheduler/cmd/scheduler.go

```diff
@@ -45,6 +49,22 @@ func (s *Scheduler) Run() {
 	}
 }
 
+func (s *Scheduler) healthCheckServer() {
+	health := func(w http.ResponseWriter, req *http.Request){
+		ctx, cancel := context.WithTimeout(req.Context(), 5*time.Second)
```

Comment:
minor: `defer cancel()` as soon as it's created allows to just forget about it, no matter how many branches can return. 

Can be cleaned up in another PR later, no rush.